docs/documentation.rst

```diff
@@ -0,0 +1,46 @@
+Generating the documentation locally
+----------------------------------
+
+Install the required packages
+
+.. code-block:: bash
+
+   $ pip install sphinx sphinx-autobuild sphinxcontrib-napoleon
+
+.. tip::
+   If you don't have sudo privileges, you can use the ``-- user`` flag to 
+   install the required packages locally.
+
+Enter in the docs directory and rename the existing conf.py
```

Comment:
My bad, I looked for instructions on how to build the documentation a while ago when I first began working on the docstrings and I couldn't find it. I didn't check again yesterday before writing the new file. Luckily it didn't take me long! I removed it.